scripts/delayed.js

```diff
@@ -5,3 +5,15 @@ import { sampleRUM } from './lib-franklin.js';
 sampleRUM('cwv');
 
 // add more delayed functionality here
+function loadUTag() {
```

Comment:
not sure how this will behave from a UX point of view. Ideally, the cookie notice shouldn't pop up randomly after 3 seconds since this would likely create a flash of content and some CLS.

Have you tried loading with async+defer from the lazy phase instead?